mq/inmemory/test.cc

```diff
@@ -147,3 +146,41 @@ TEST(InMemoryMQ, WaitOnOverflowTest) {
   std::set<std::string> messages(begin(c.messages_), end(c.messages_));
   EXPECT_EQ(messages.size(), 100u);
 }
+
+TEST(InMemoryMQ, DefaultConsumer) {
+  struct Storage {
+    std::string messages;
+    size_t count = 0u;
```

Comment:
I think this should be `atomic_size_t` (or `volatile`, but that would be more confusing).